openmc/material.py

```diff
@@ -973,3 +973,73 @@ def export_to_xml(self, path='materials.xml'):
         # Write the XML Tree to the materials.xml file
         tree = ET.ElementTree(root_element)
         tree.write(path, xml_declaration=True, encoding='utf-8')
+
+
+def make_boric_acid(boron_ppm, temperature=293., pressure=0.1013, density=None,
+                    **kwargs):
+    """Return a Material with the composition of boric acid.
+
+    The water density can either be given directly, or it can be determined from
+    a temperature and pressure.
+
+    Parameters
+    ----------
+    boron_ppm : float
+        The weight fraction in parts-per-million of elemental boron in the acid.
+    temperature : float
+        Water temperature in [K] used to compute water density.
+    pressure : float
+        Water pressure in [MPa] used to compute water density.
```

Comment:
Now that I see the structure more, perhaps this function should be the one to have an optional pressure units input to allow for psia and `water_density` can be left in terms of MPa

Reply:
Next you'll want me to give you flux with per square inches!  But you're right.  This function is a good candidate to support multiple units.